scripts/setup/upgrade-postgres

```diff
@@ -40,6 +40,7 @@ fi
 
 # Capture the output so we know where the path to the post-upgrade scripts is
 UPGRADE_LOG=$(mktemp "/var/log/zulip/postgres-upgrade-$UPGRADE_FROM-$UPGRADE_TO.XXXXXXXXX.log")
+set -o pipefail
```

Comment:
A global directive like this should probably go at the top, and be combined with `set -e` (`set -eo pipefail`).

Reply:
Sure, makes sense.  I think it obscures which it's relevant to a little, but I'm sympathetic to the fact that putting it at the top making clear that it's global.

Reply:
Perhaps ideally we’d have a lint-enforced policy of using `bash -euo pipefail` for all scripts, and we wouldn’t need to worry about which individual commands it’s relevant for. That’d take some auditing work, though.